src/device/controllers/game_controller.c

```diff
@@ -118,9 +118,10 @@ static void pak_read_block(struct game_controller* cont,
 
     if (cont->ipak != NULL) {
         cont->ipak->read(cont->pak, address, data, PAK_CHUNK_SIZE);
+        *dcrc = pak_data_crc(data, PAK_CHUNK_SIZE);
+    } else {
+        *dcrc = ~pak_data_crc(data, PAK_CHUNK_SIZE);
```

Comment:
I suggest to add a simple comment after this line: `// dont checksum when mempak is not present`.

Those six words would make the branching intent better.

Reply:
ok I;ve added a comment explaining the code